nmdc_server/models.py

```diff
@@ -273,18 +273,6 @@ def open_in_gold(self) -> Optional[str]:
             self.gold_study_identifiers,  # type: ignore
         )
 
-    @property
-    def award_dois(self) -> list[DOIInfo]:
-        return [d for d in self.dois if d.doi_type == DOIType.AWARD]  # type: ignore
-
-    @property
-    def publication_dois(self) -> list[DOIInfo]:
-        return [d for d in self.dois if d.doi_type == DOIType.PUBLICATION]  # type: ignore
-
-    @property
-    def dataset_dois(self) -> list[DOIInfo]:
-        return [d for d in self.dois if d.doi_type == DOIType.DATASET]  # type: ignore
-
     @property
     def doi_map(self) -> Dict[str, Any]:
```

Comment:
Related to the above changes, wouldn't the construction of the `doi_map` need to be updated now that there aren't `self.<doi_type>_dois` properties?

Reply:
Yes, you are right. I though I had updated it I think I might have lost that on rebase. I'll either recover or re-write it

Reply:
fixed in 2a7c9385d07f653b2cd5c2d902529fac30c6f2f0